frontend/src/pages/OrderPayListPage.tsx

```diff
@@ -0,0 +1,15 @@
+import Title from "../components/ui/Title";
+import OrderPayList from "../components/order-pay-list/OrderPayList";
+import OrderPaySection from "../components/order-pay-list/OrderPaySection";
+
+export default function OrderPayListPage() {
+  return (
+    <section className="order-section">
+      <Title variant="order" text="주문/결제"></Title>
```

Comment:
self closing tag를 사용해보세용!
```suggestion
      <Title variant="order" text="주문/결제" />
```